cv.typ

```diff
@@ -1,298 +1,348 @@
+    }
+
+    // Add any social profiles
+    #if info.personal.profiles.len() > 0 {
+        for profile in info.personal.profiles {
+            profiles.push(
+                box(link(profile.url)[#profile.url.split("//").at(1)])
+            )
         }
+    }
 
-// Arrange the contact profiles with a diamond separator
-#let contacttext = block(width: 100%)[
     // #set par(justify: false)
-    #set text(font: bodyfont, weight: "medium", size: fontsize * 1)
+    #set text(font: uservars.bodyfont, weight: "medium", size: uservars.fontsize * 1)
     #pad(x: 0em)[
         #profiles.join([#sym.space.en #sym.diamond.filled #sym.space.en])
     ]
 ]
 
 // Create layout of the title + contact info
-#align(center)[
-    = #info.personal.name
-    #addresstext
-    #contacttext
-    // #v(0.5em)
+#let cvheading(info, uservars) = [
+    #align(center)[
+        = #info.personal.name
+        #addresstext(info, uservars)
+        #contacttext(info, uservars)
+        // #v(0.5em)
+    ]
 ]
 
 // Education
-== Education
-
-#for edu in info.education [
-    // Parse ISO date strings into datetime objects
-    #let start = utils.strpdate(edu.startDate)
-    #let end = utils.strpdate(edu.endDate)
-
-    // Create a block layout for each education entry
-    #block(width: 100%)[
-        // Line 1: Institution and Location
-        *#link(edu.url)[#edu.institution]* #h(1fr) *#edu.location* \
-        // Line 2: Degree and Date Range
-        #text(style: "italic")[#edu.studyType in #edu.area] #h(1fr)
-        #utils.monthname(start.month()) #start.year() #sym.dash.en #utils.monthname(end.month()) #end.year() \
-        // Bullet points
-        - *Honors*: #edu.honors.join(", ")
-        - *Courses*: #edu.courses.join(", ")
-        // Highlights or Description
-        #for hi in edu.highlights [
-            - #eval("[" + hi + "]")
+#let cveducation(info) = [
+    #if info.education != none [
+        == Education
+
+        #for edu in info.education [
+            // Parse ISO date strings into datetime objects
+            #let start = utils.strpdate(edu.startDate)
+            #let end = utils.strpdate(edu.endDate)
+
+            // Create a block layout for each education entry
+            #block(width: 100%)[
+                // Line 1: Institution and Location
+                *#link(edu.url)[#edu.institution]* #h(1fr) *#edu.location* \
+                // Line 2: Degree and Date Range
+                #text(style: "italic")[#edu.studyType in #edu.area] #h(1fr)
+                #utils.monthname(start.month()) #start.year() #sym.dash.en #utils.monthname(end.month()) #end.year() \
+                // Bullet points
+                - *Honors*: #edu.honors.join(", ")
+                - *Courses*: #edu.courses.join(", ")
+                // Highlights or Description
+                #for hi in edu.highlights [
+                    - #eval("[" + hi + "]")
+                ]
+            ]
         ]
     ]
 ]
```

Comment:
Sorry, I meant for example at `cveducation(info)` the part `cveducation(info) = [...]` could be `cveducation(info) = {...}` instead and for similar functions.

Reply:
I have pushed a commit to address this. Could you please take a look on it?